modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -74,6 +74,7 @@ GrainTracker::GrainTracker(const InputParameters & parameters)
     _poly_ic_uo(parameters.isParamValid("polycrystal_ic_uo")
                     ? &getUserObject<PolycrystalUserObjectBase>("polycrystal_ic_uo")
                     : nullptr),
+    _verbosity_level(getParam<short>("verbosity_level")),
```

Comment:
`_short_`  sometime just does not work. It is better to just use `int`

Reply:
No, it **always** works. Do you have a counter example? I use `short` in several places in the framework. I'd probably use `char` except those don't print nicely by default 😄

Reply:
@fdkong - I just verified that I can override the parameter from the command line with all three values (0, 1, and 2) and it all works fine.